third_party/nvfuser/test/test_gpu_fused_reduction.cpp

```diff
@@ -1219,12 +1219,18 @@ TEST_F(NVFuserTest, FusionGroupAllreduce5_CUDA) {
   auto tv10 = broadcast(tv9, {true});
   auto tv11 = div(tv8, tv10);
 
-  auto out = add(
-      add(castOp(DataType::Double, tv3), tv7), castOp(DataType::Double, tv11));
+  auto tv12 = makeSymbolicTensor(1, DataType::ComplexFloat);
+  fusion.addInput(tv12);
+  auto tv13 = sum(tv12, {0});
+  auto tv14 = broadcast(tv13, {true});
+  auto tv15 = div(tv12, tv14);
 
+  auto outFDI = add(
+      add(castOp(DataType::Double, tv3), tv7), castOp(DataType::Double, tv11));
+  auto out = add(outFDI, castOp(DataType::Double, tv15));
```

Comment:
Should we cast to complex double here?

Reply:
this is just pick the real part and cast to double. All results from other data types are casted to double, so does the reference results from torch.

Reply:
Should we cast all results from other data types and reference results to complex double as well? Casting to double discard imag so we are not checking the correctness of imag.

Reply:
updated this test with all results converted to complex<double>